[doc] Glossary +SEH +TFM (#51878)
dotnet/runtime: docs/project/glossary.md

| ProjectN | Codename for the first version of [.NET Native for UWP](https://msdn.microsoft.com/en-us/vstudio/dotnetnative.aspx). |
| R2R | Ready-to-Run. A flavor of native images - command line switch of [crossgen](../workflow/building/coreclr/crossgen.md). |
| Redhawk | Codename for experimental minimal managed code runtime that evolved into [CoreRT](https://github.com/dotnet/corert/). |

> | SEH | Structured Exception Handling - Microsoft extension to C see [details](https://docs.microsoft.com/en-us/cpp/cpp/structured-exception-handling-c-cpp) |

Reviewer: Use the culture-agnostic link to the SEH docs.

```suggestion
| SEH | Structured Exception Handling - Microsoft extension to C see [details](https://docs.microsoft.com/cpp/cpp/structured-exception-handling-c-cpp) |
```

> | SEH | Structured Exception Handling - Microsoft extension to C see [details](https://docs.microsoft.com/en-us/cpp/cpp/structured-exception-handling-c-cpp) |

Reviewer 2: ```suggestion
| SEH | [Structured Exception Handling](https://docs.microsoft.com/en-us/windows/win32/debug/structured-exception-handling). Unified mechanism for handling hardware and software exceptions on Windows. |
```
SEH is Windows OS feature. The compiler extension merely exposes it.

> | SEH | [Structured Exception Handling](https://docs.microsoft.com/en-us/windows/win32/debug/structured-exception-handling). Unified mechanism for handling hardware and software exceptions on Windows. |

Reviewer 2: ```suggestion
| SEH | [Structured Exception Handling](https://docs.microsoft.com/windows/win32/debug/structured-exception-handling). Unified mechanism for handling hardware and software exceptions on Windows. |
```

| SOS | [Son of Strike](https://docs.microsoft.com/en-us/archive/blogs/jasonz/sos-debugging-of-the-clr-part-1). The debugging extension for DbgEng based debuggers. Uses the DAC as an abstraction layer for its operation. |
| SuperPMI | JIT component test framework (super fast JIT testing - it mocks/replays EE in EE-JIT interface) - see [SuperPMI details](https://github.com/dotnet/runtime/blob/main/src/coreclr/ToolBox/superpmi/readme.txt). |
| SVR | The CLR used to be built as two variants, with one called "mscorsvr.dll", to mean the "server" version. In particular, it contained the server GC implementation, which was intended for multi-threaded apps capable of taking advantage of multiple processors. In the .NET Framework 2 release, the two variants were merged into "mscorwks.dll". The WKS version was the default, however the SVR version remained available. |

> | TFM | [Target Framework Moniker](https://docs.microsoft.com/en-us/dotnet/standard/frameworks) such as `net6.0` or `wpa81`. |

Reviewer 2: ```suggestion
| TFM | [Target Framework Moniker](https://docs.microsoft.com/en-us/dotnet/standard/frameworks) such as `net6.0` or `netstandard2.0`. |
```
wpa81 is a very obscure TFM. I would be better to use the more common one as an example.

> | TFM | [Target Framework Moniker](https://docs.microsoft.com/en-us/dotnet/standard/frameworks) such as `net6.0` or `netstandard2.0`. |

Reviewer 2: ```suggestion
| TFM | [Target Framework Moniker](https://docs.microsoft.com/dotnet/standard/frameworks) such as `net6.0` or `netstandard2.0`. |
```
